sway-book/README.md

```diff
@@ -0,0 +1,28 @@
+# Setup
+
+> TODO: forc plugins etc. need to be added back in
+
+If you wish to alter the documentation presented in this book then follow the instructions.
+
+1. Install [Rust](https://www.rust-lang.org/tools/install) if it's not installed.
+2. Install [mdbook](https://rust-lang.github.io/mdBook/).
+
+   ```bash
+   cargo install mdbook
+   ```
+
+3. To [build](https://rust-lang.github.io/mdBook/cli/build.html) the book make sure that you are in `/sway/sway-book` and run
+
+   ```bash
+   mdbook build
```

Comment:
I have Rust and `mdbook` installed but I get this error message at this step:
```
$ mdbook build
2022-11-01 14:33:09 [INFO] (mdbook::book): Book building has started
2022-11-01 14:33:09 [WARN] (mdbook::preprocess::cmd): The command wasn't found, is the "forc-documenter" preprocessor installed?
2022-11-01 14:33:09 [WARN] (mdbook::preprocess::cmd): 	Command: mdbook-forc-documenter
2022-11-01 14:33:09 [ERROR] (mdbook::preprocess::links): Error updating "{{#include ../../../../examples/msg_sender/src/main.sw}}", Could not read file for link {{#include ../../../../examples/msg_sender/src/main.sw}} (/Users/anton/fuel.network/sway/sway-book/src/documentation/operations/call-data/../../../../examples/msg_sender/src/main.sw)
2022-11-01 14:33:09 [WARN] (mdbook::preprocess::links): Caused By: No such file or directory (os error 2)
2022-11-01 14:33:09 [ERROR] (mdbook::preprocess::links): Error updating "{{#include ../../../examples/ownership/src/main.sw:set_owner_example}}", Could not read file for link {{#include ../../../examples/ownership/src/main.sw:set_owner_example}} (/Users/anton/fuel.network/sway/sway-book/src/documentation/examples/../../../examples/ownership/src/main.sw)
2022-11-01 14:33:09 [WARN] (mdbook::preprocess::links): Caused By: No such file or directory (os error 2)
2022-11-01 14:33:09 [ERROR] (mdbook::preprocess::links): Error updating "{{#include ../../../examples/ownership/src/main.sw:revoke_owner_example}}", Could not read file for link {{#include ../../../examples/ownership/src/main.sw:revoke_owner_example}} (/Users/anton/fuel.network/sway/sway-book/src/documentation/examples/../../../examples/ownership/src/main.sw)
2022-11-01 14:33:09 [WARN] (mdbook::preprocess::links): Caused By: No such file or directory (os error 2)
2022-11-01 14:33:09 [INFO] (mdbook::book): Running the html backend
```

Reply:
Does it prevent the book from building? I think this should allow you to open the book in the browser.

Those links are pointing at the current examples in the live sway book. I have been moving things around and commenting things out so I have not updated all of the links / re-created new examples so if there is a link that does not point at a file then it will show a warning, or an error. I usually ignore them because I can use the book and over time get to the places where I need to update the sections.

If this prevents the book from building then let me know otherwise this can be ignored.